common/app/model/CardStylePicker.scala

```diff
@@ -9,19 +9,23 @@ import commercial.targeting.CampaignAgent
 
 object CardStylePicker {
 
-  def apply(content: CapiContent, trailMetaData: MetaDataCommonFields): CardStyle = {
+  def apply(content: CapiContent): CardStyle = {
     val tags = content.tags.map(_.id).toSeq
     extractCampaigns(tags) match {
       case Nil => CardStyle(content, TrailMetaData.empty)
-      case _   => SpecialReport
+      case campaign :: _ =>
```

Comment:
When the campaign id is "SpecialReportAlt" facia-press decides that CardStyle will be SpecialReportAlt.